### ISSUE TITLE
Test/check function

### ISSUE THREAD

**Issue by Petros626** [lws803/BLE-jammer#5]
Hello,

I have adapted the code for my Arduino Nano Every (GPIO pins) and adopted the software except for 
`byte text = 255;` -> `uint8_t text = "x";` (it's the same datatype)

In the description was the character x, so I chose this. 255 in ASCII would be "ÿ"  (actually the character doesn't matter). The problem is that, After I connected all modules and wanted to test whether really no new devices are found, I found however that my smartphone finds sometimes one then again not (am not so convinced).

Questions:

1. the jammer can only block devices unknown to the smartphone from being visible, already known devices are connected without any problems.

2. is there a code to test if the RF modules work. Have read in some reviews that you already arrived defective. How did you test if the modules work?

Greetings,

 Petros

**Comment by lws803** [comment]
Hi there @Petros626,

Thanks for taking the time to try this out and apologies for the slow response on this, had a pretty busy week. To answer your questions:

1. That is correct, this jammer will only block devices from discovering each other. If you want to block devices from communicating entirely even after discovery, you will have to block all channels instead of just the 3 proposed by this repository.
2. I tested it by using an RF spectrum analyzer. An [RF explorer](https://j3.rf-explorer.com/) should do the job but it can be pretty pricey. Alternatively, you could also use a BLE app on your smartphone and program an **ESP32 Development Board with Wireless Module** to broadcast in BLE. This device should not be discoverable by your smartphone if the jammer is active.

**Comment by Petros626** [comment]
Hello @lws803 ,

thank you for answering.

So I was able to test my three modules one by one with a specific code example from the `RF24 `library. They're all working without doubts. So the important part are these 4 params of the function: `printDetails() `debugging information were:

`EN_AA =0x3f`
`EN_RXADRR =0x02`
`RF_CH =0x4c`
`RF_SETUP =0x03`
`CONFIG =0x0f`

So I validated, that the modules aren't broken. 

For now I' using your code I get different addresses for the params and wanted to ask you, if you get the same results for your three modules. The `EN_AA` (autoacknowledge) is fine, I guess, because it`s FALSE in the Code and `RF_CH `differs for the 3 frequencies. All in all it should work :) ?


`nRF24L01 module :`

`EN_AA		= 0x00`
`EN_RXADDR	= 0x03`
`RF_CH		= 0x02`
`RF_SETUP	= 0x0d`
`CONFIG		= 0x0e`


`nRF24L01 module 2:`

`EN_AA		= 0x0`
`EN_RXADDR	= 0x03`
`RF_CH		= 0x1a`
`RF_SETUP	= 0x0d`
`CONFIG		= 0x0e`


`nRF24L01 module 3:`


`EN_AA		= 0x00`
`EN_RXADDR	= 0x03`
`RF_CH		= 0x50`
`RF_SETUP	= 0x0d`
`CONFIG		= 0x0e`
